examples/autocomplete_path.rs

```diff
@@ -0,0 +1,84 @@
+use inquire::{CustomUserError, Text};
+
+fn main() {
+    let ans = Text::new("Profil picture:")
+        .with_suggester(&suggest_file_paths)
+        .with_completer(&complete_file_path)
+        .prompt();
+
+    match ans {
+        Ok(path) => println!("Path: {path}"),
+        Err(error) => println!("Error with questionnaire, try again later: {error:?}"),
```

Comment:
Btw, could we do this?
```rust
        Ok(path) => println!("Path: {}", path),
        Err(error) => println!("Error with questionnaire, try again later: {:?}", error),
```

Even though this is on an example and in theory it doesn't break the lib's MSRV, it will be kind of a pain to change the current GH workflow file to compile this file as one of the tests is under 1.53.0 and this feature was stabilized recently (I actually had to update my toolchain to compile the example lol)

Reply:
No problem. I'm doing this right now. I didn't think to that sorry :wink: